documentation/modules/upgrading/con-upgrade-cluster.adoc

```diff
@@ -0,0 +1,102 @@
+// This module is included in the following assemblies:
+//
+// upgrading/assembly-upgrade.adoc
+
+[id='con-upgrade-cluster-{context}']
+= Upgrading Kubernetes
+
+[role="_abstract"]
+If you are upgrading Strimzi, you may also need to upgrade your Kubernetes cluster to a supported version.
+
+Refer to your Kubernetes distribution upgrade documentation to check the upgrade path and the steps to upgrade your nodes correctly.
+
+NOTE: Before upgrading, {supported-configurations}[check supported Kubernetes versions^].
+
+When performing your upgrade, you'll want to keep your Strimzi clusters available.
+You can employ the following strategies to do this.
+
+* Configuring pod disruption budgets
+* Updating pods while keeping topics available
+
+== Configuring pod disruption budgets
+
+A pod disruption budget allows only a specified number of pods to be unavailable at a given time.
+During planned maintenance of Kafka broker pods, a pod disruption budget ensures Kafka continues to run in a highly available environment.
+
+You specify a pod disruption budget using a `template` customization for a Kafka component.
+Labels and annotations identify resources.
+By default, pod disruption budgets allow only a single pod to be unavailable at a given time.
+You can increase the number of unavailable pods allowed by changing the default value of the `maxUnavailable` property.
+
+If you used the Cluster Operator to deploy Kafka, you can use the Strimzi Drain Cleaner tool to evict nodes during an upgrade.
+To do this, you set `maxUnavailable` to 0. This prevents voluntary disruptions, so pods must be evicted manually.
+
+.Specifying a pod discruption budget
+[source,yaml,subs=attributes+]
+----
+# ...
+template:
+  podDisruptionBudget:
+    metadata:
+      labels:
+        key1: label1
+        key2: label2
+      annotations:
+        key1: label1
+        key2: label2
+    maxUnavailable: 1
+# ...
+----
+
+== Updating pods while keeping topics available
+
+During an upgrade, you can trigger a rolling update of pods through the Cluster Operator.
+Using `Pod` resources, rolling updates restart the pods of resources with new pods.
```

Comment:
We should stress out here that the user needs to watch which pods need to be drained => and then annotate the correct pod.